website/src/index.html

```diff
@@ -0,0 +1,210 @@
+                <img src="./images/daysoff/5.png" alt="Time to leave waiver manager" class="daysoff__animation" data-daysoff="5">
+                <img src="./images/daysoff/6.png" alt="Time to leave waiver manager" class="daysoff__animation" data-daysoff="6">
+                <img src="./images/daysoff/7.png" alt="Time to leave waiver manager" class="daysoff__animation" data-daysoff="7">
+                <img src="./images/daysoff/8.png" alt="Time to leave waiver manager" class="daysoff__animation" data-daysoff="8">
+                <img src="./images/daysoff/9.png" alt="Time to leave waiver manager" class="daysoff__animation" data-daysoff="9">
+                <img src="./images/daysoff/10.png" alt="Time to leave waiver manager" class="daysoff__animation" data-daysoff="10">
+                <img src="./images/daysoff/11.png" alt="Time to leave waiver manager" class="daysoff__animation" data-daysoff="11">
+                <img src="./images/daysoff/12.png" alt="Time to leave waiver manager" class="daysoff__animation" data-daysoff="12">
+                <img src="./images/daysoff/13.png" alt="Time to leave waiver manager" class="daysoff__animation" data-daysoff="13">
+                <img src="./images/daysoff/14.png" alt="Time to leave waiver manager" class="daysoff__animation" data-daysoff="14">
+                <img src="./images/daysoff/15.png" alt="Time to leave waiver manager" class="daysoff__animation" data-daysoff="15">
+            </div>
+
+            <div class="marketing js-daysoff--trigger">
+                <div class="marketing__container js-reveal">
+                    <h2 class="marketing__title">Add a Waiver for days off</h2>
+                    <p class="marketing__desc">You can also add waivers on the days you did not work.</p>
+                </div>
+            </div>
+        </section>
+
+
+        <section class="cta">
+
+            <h1 class="intro__title js-reveal">
+                Time to Leave
+                <img src="images/resources/timer.svg" alt="Time To Leave Logo">
+            </h1>
+            <p class="cta__desc js-reveal">
+                It is highly customizable, not only on the look and feel (with different month and day visions,
+                different themes, and more), but also offers lots of functionalities such as defining how many hours you
+                usually work per day, which week days, allow you to quickly source holidays based on your region
+                (Country, State, and City), and much more.
+            </p>
+            <div class="cta__free js-reveal">
+                <h3 class="cta__free__money">FREE!</h3>
+                <p class="cta__free__desc">Personal Time Management</p>
+
+                <h4 class="cta__free__punchline">No more guessing</h4>
+                <a href="https://github.com/thamara/time-to-leave/releases/tag/v.1.5.5" class="button--primary">Download</a>
```

Comment:
```suggestion
                <a href="https://github.com/thamara/time-to-leave/releases/latest" class="button--primary">Download</a>
```

We should probably link to the latest release, so we can avoid having to update this every time.

Reply:
This is a perfect idea. Is there something similar for the MacOS, Windows & Linux version links in the dropdown?